src/interfaces/jira.ts

```diff
@@ -192,6 +192,49 @@ export interface JiraRemoteLinkStatus {
 	label: string;
 }
 
+export interface JiraVulnerabilityBulkSubmitData {
+	vulnerabilities: JiraVulnerability[];
+}
+
+export interface JiraVulnerability {
+	id: string;
+	schemaVersion: string;
+	updateSequenceNumber: number;
+	containerId: string;
+	displayName: string;
+	description: string;
+	url: string;
+	type: string;
```

Comment:
What do you think about also making this an enum with values `sca, sast, dast, unknown`?